storage/src/main/java/org/apache/uniffle/storage/handler/impl/LocalFileWriteHandler.java

```diff
@@ -52,17 +52,11 @@ public LocalFileWriteHandler(
 
   private void createBasePath() {
     File baseFolder = new File(basePath);
-    // check if shuffle folder exist
-    if (!baseFolder.exists()) {
-      try {
-        // try to create folder, it may be created by other Shuffle Server
-        baseFolder.mkdirs();
-      } catch (Exception e) {
```

Comment:
Only `SecurityException` will be thrown when access is denied by `System.getSecurityManager()`.